user/README.md

```diff
@@ -0,0 +1,94 @@
+```bash
```

Comment:
@MaazMS what is this file?

Reply:
by mistake this file is added. I will remove this file